internal/testhelpers/sql.go

```diff
@@ -26,21 +26,34 @@ func CleanSQL(t *testing.T, c *pop.Connection) {
 
 		new(login.FlowMethods).TableName(),
 		new(login.Flow).TableName(),
+		"selfservice_login_request_methods",
+		"selfservice_login_requests",
```

Comment:
I had to add all of these tables because rerunning the tests locally did not work otherwise. Am I missing something?

Reply:
Hm they should be removed by the migration itself. I don't think they need to be added?

Reply:
I get a `table already exists` error without adding them here. `CleanSQL` is not running down migrations but drops all tables, seems like some were missing here. Or what do you mean by "removed by the migration itself"?

Reply:
Is `CleanSQL` called for the migration tests? If so, I think these need to stay. What I meant was that these tables are renamed/removed by the SQL Migrations (they are renamed to `selfservice_login_flows` for example).

Reply:
Ok could not reproduce with clean databases. I probably killed the test in the middle of migrations at some point.